examples/pretrain_finetune_huggingface.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>link is broken in NBViewer... is it on renderer or genuine bug?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosaicml/composer/pull/1775/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
The link works for me... https://app.reviewnb.com/mosaicml/composer/pull/1775?

Reply:
<img width="1442" alt="image" src="https://user-images.githubusercontent.com/17102158/205358340-13083d59-1ca4-476a-9126-be3143ee1b8d.png">
this is what I see -- should there be links here?

Reply:
oh thats what you meant. yeah those appear to be broken in nbviewer, they are ok for me locally